### DISCUSSION TITLE
Will having the same face in 4 or 5 concept images make the same face show up all the time in outputs?

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#1319] [Q&A]
Im fairly sure this will happen in instance images. can you tell me how to avoid that? I wanna have some faces in instance images. if i have strictly different faces for all instance images, will that prevent all outputs having the same face from instance images?

**Comment by cdefghijkl** [comment]
you can write different filename for each face. I did it with 4 different person. Sometimes the generated result will be little similar but only some features like the mole or jaw or ears.
